### ISSUE TITLE
Weight decay and BN

### ISSUE THREAD

**Issue by lolemacs** [torch/torch7#846]
I'm trying to reproduce a few papers that were implemented in Torch, but it's unclear to me how weight decay works on Torch. Does it also penalize Batch Norm params (gamma and beta)?

For example, one of the repos I'm trying to reproduce is:
https://github.com/szagoruyko/wide-residual-networks

**Comment by soumith** [comment]
Yes it does.
